### PR TITLE
Update terraform.config to 7.0

### PR DESCRIPTION
**Pull request by peteruithoven**
Fixes: #655

### CODE REVIEW

File: etc/terraform.conf [hunk 1]
```diff
@@ -30,7 +30,7 @@ HWE_X11="no"
 INCLUDE_APPCENTER=""
 
 # suffix for generated .iso files
-OUTPUT_SUFFIX=""
+OUTPUT_SUFFIX="rc"
```
Comment (vjr) [inline etc/terraform.conf:33]: Is the RC suffic necessary now?

Comment (peteruithoven) [reply etc/terraform.conf:33]: Good question, I've asked it on Slack
https://ele.slack.com/archives/C8JPLKKQQ/p1676473060941199

Comment (peteruithoven) [reply etc/terraform.conf:33]: @tintou response: 
> No, we are not at RC stage anymore 

So I'll update this.

**Comment by peteruithoven** [comment]
Switched to non azure mirror as was recommended here:
https://github.com/elementary/os/issues/655#issuecomment-1431617601

**Review by davidmhewitt — APPROVED** [review]

**Comment by davidmhewitt** [comment]
Thank you!

**Pull request merged**